### PR TITLE
add css colors in globals

### LINKED ISSUE #10: add css colors in globals

### PR DESCRIPTION
**Pull request by mewiemers**
Fixes #10

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -3,6 +3,16 @@
   src: local("AlmondCream"), url(../public/AlmondCream.woff) format("woff");
 }
 
+:root {
+  --mainbackground-color: #50c4b7;
```
Comment (fre-ben) [inline styles/globals.css]: Maybe name this `--main-background-color` or `--primary-color`, because there is a secondary color in line 8?

**Review by fre-ben — COMMENTED** [review]
lgtm 🚀

**Review by RonjaMeinecke — APPROVED** [review]
LGTM

File: components/welcome/Welcome.tsx [hunk 2]
```diff
@@ -7,12 +7,11 @@ function Logo() {
   return (
     <div className={styles.background}>
       <h1 className={styles.headline}>Meet</h1>
-      <Link href="#">
+      <Link href="/">
```
Comment (lmachens) [inline components/welcome/Welcome.tsx]: I think this is should not be part of this PR. See https://github.com/mewiemers/meet-albert/pull/11

File: styles/globals.css [hunk 3]
```diff
@@ -3,12 +3,28 @@
...
+  font-family: "Gabriola";
+  src: local("Gabriola"), url(../public/Gabriola.woff) format("woff");
+}
+
+:root {
+  --mainbackground-color: #50c4b7;
+  --secondary-color: #fff;
+  --link-color: #674b14;
+  --font-color: #000;
+  --quizbutton-color: #2537c7;
```
Comment (lmachens) [inline styles/globals.css]:
```suggestion
  --quiz-button-color: #2537c7;
```

File: styles/globals.css [hunk 4]
```diff
@@ -3,12 +3,28 @@
...
+}
+
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue,
+    "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
```
Comment (lmachens) [inline styles/globals.css]: `Franklin Gothic Medium` and `Arial Narrow` are not imported.
And it is very unlikely that it is used because of this long list of font families.

**Review by lmachens — COMMENTED** [review]
gj!
